client/src/components/auth/SignUp.js

```diff
@@ -40,9 +40,27 @@ export default class SignUp extends Component {
     }
 
     try {
-      await this.props.login()
-    } catch (error) {
-      console.log(error.message)
+      const requestOptions = {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json', 'Access-Control-Allow-Origin': "*" },
+        body: JSON.stringify({ username: this.state.name, email: this.state.email, password: this.state.password })
+      };
+
+      fetch('http://localhost:3002/users', requestOptions)
+          .then(res => {
+            if (res.status === 201){
+              fetch('http://localhost:3002/users/login', requestOptions)
```

Comment:
You shouldn't have to make another call to login here, instead change the signup route on https://github.com/hatchways/team-fettuccini/blob/feature-auth-integration/server/routes/users.js so that it sets the token on the cookies (like on the login route) instead of returning it on the body of the response